yang/frr-bgp-common-multiprotocol.yang

```diff
@@ -204,5 +204,25 @@ submodule frr-bgp-common-multiprotocol {
       description
         "IPv6 flowspec configuration options.";
     }
+
+    container linkstate-linkstate {
+      when "derived-from-or-self(../afi-safi-name, 'frr-rt:linkstate-linkstate')" {
+        description
+          "Include this container for Link-State specific
+           configuration.";
+      }
+      description
+        "Link-State configuration options.";
+    }
+
+    container linkstate-linkstate-vpn {
+      when "derived-from-or-self(../afi-safi-name, 'frr-rt:linkstate-linkstate-vpn')" {
+        description
+          "Include this container for Link-State VPN specific
+           configuration.";
+      }
+      description
+        "Link-State VPN configuration options.";
+    }
```

Comment:
as we will not cover the CLI for linkstate-vpn, should we really cover linkstate-vpn yang mode ?

Reply:
removed